### ISSUE TITLE
Breaking change introduced by v1.0.6 - removal of `NewAnalyzer`

### ISSUE THREAD

**Issue by jamietanna** [kunwardeep/paralleltest#22]
In https://github.com/kunwardeep/paralleltest/commit/8da1323054fee1167a31e061738c6408e42e3341 we've removed the `NewAnalyzer` method which is used by other tools, such as golangci-lint

**Comment by ldez** [comment]
I don't think there is a problem with the method `NewAnalyzer`, I think you are not using the right version of paralleltest.

**Comment by jamietanna** [comment]
Per convo on the golangci-lint issue, it looks like it's been worked around in https://github.com/golangci/golangci-lint/commit/97eea6ea4921f4594b917b38fae52506a46ea018 but this still was a breaking change, so thought it'd be good to flag it 👍

**Comment by ldez** [comment]
> but this still was a breaking change, so thought it'd be good to flag it

Yes, semver should be followed, it's important to inform lib users and it's really important with Go modules.

**Comment by kunwardeep** [comment]
Thanks for bringing this up. But is it okay to close this issue for now?
